Library/Formula/libmemcached.rb

```diff
@@ -15,7 +13,11 @@ class Libmemcached < Formula
   end
 
   option "with-sasl", "Build with sasl support"
+  option "with-static", "Build static library"
+  option "with-memaslap", "Build with load generation and benchmark tool support"
+  option "with-hsieh-hash", "Build with hsieh hashing support"
 
+  depends_on "libevent" # requires --enable-libmemcachedprotocol
```

Comment:
@DomT4 Example here https://github.com/php-memcached-dev/php-memcached/pull/98 if it is missing it too will only work without additional options.